Dockerfile.base

```diff
@@ -38,6 +38,7 @@ RUN apt-get update && \
     locales \
     python3-pip \
     dumb-init \
+  && update-alternatives --install /usr/bin/python python /usr/bin/python3 10 \
```

Comment:
in the python community its generally accepted that `python` is whatever the OS decides is default and `$ python3` is specific to python 3. Linking the two is a bad practice as python2 is still generally used (albeit not under best cases) and I don't intend to introduce that confusion here

Reply:
Understood.  Thanks for taking a look.